src/index.html

```diff
@@ -4,13 +4,43 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, 
+      maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Search bar airbnb</title>
+    <title>Search bars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="search-container">
+      <form 
+        class="search-box" 
+        data-qa="big"
+      >
+        <label for="search-input"></label>
+        <input 
+          type="text" 
+          id="search-input" 
+          class="search-input" 
+          placeholder="&nbsp;&nbsp;Try &nbsp;&quot;Los Angeles&quot;"
```

Comment:
```suggestion
          placeholder="Try &quot;Los Angeles&quot;"
```
You don't need hardcore spaces. Just add some margins or padding if needed